BUILDING.md

```diff
@@ -178,3 +178,30 @@ npm start
 Then, open a browser to http://localhost:8080.
 
 [^1]: Oxford English Dictionary
+
+## Running Project Leo in a Docker Container
```

Comment:
How about making this as a third option under "## Running Project Leo"?

- Change "There are two ways to run Project Leo:" to "Project Leo can be run in the following ways:"
- Move this section before "[^1] ...", that should stay at the bottom of the page.
- Change the header from ## to ###.